package.json

```diff
@@ -16,9 +16,9 @@
   "bugs": {
     "url": "http://github.com/nock/nock/issues"
   },
-  "engines": [
-    "node >= 4.0"
-  ],
+  "engines": {
+    "node": ">= 6.0"
+  },
```

Comment:
That will release a new 10.x fix version, and we shouldn't drop Node 4 support in v10

Reply:
Gotcha, I didn't realize 4.x was supported.

Reply:
ooh my fault, I mixed up projects lol, sorry! We don't support Node 4 any more in v10.x